src/django_upgrade/fixers/management_command.py

```diff
@@ -0,0 +1,44 @@
+"""
+Replace BaseCommand.requires_system_checks boolean flag by list of checks:
+https://docs.djangoproject.com/en/stable/releases/3.2/#deprecated-features-3-2
+"""
+from __future__ import annotations
+
+import ast
+from functools import partial
+from typing import Iterable
+
+from tokenize_rt import Offset
+
+from django_upgrade.ast import ast_start_offset
+from django_upgrade.data import Fixer, State, TokenFunc
+from django_upgrade.tokens import replace
+
+fixer = Fixer(
+    __name__,
+    min_version=(3, 2),
+)
+
+REPLACEMENTS = {
+    True: '"__all__"',
+    False: "[]",
+}
```

Comment:
I think this would be clearer just as an `if` statement inside the function, there aren't any othre cases that may be added in the future